client/sources/ok_test/scheme.py

```diff
@@ -0,0 +1,96 @@
+"""Console for interpreting Scheme. In particular, this is meant to integrate
+with UC Berkeley CS 61A's Scheme project. SchemeConsole expects the an
+importable module called "scheme" with the following interface:
+
+    scheme.create_global_frame()
+    scheme.read_eval_print_loop(next_line_fn, frame, interactive)
+    scheme.buffer_input()
+    scheme.read_line(code)
+    scheme.scheme_eval(exp, env)
```

Comment:
We should toss in a link to the scheme project so that it's obvious what these should do.